cves/CVE-2017-14058.yml

```diff
@@ -64,7 +64,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: 
+  A read function in ffmpeg does not restrict reload attempts for an insufficient 
+  list, so a denial of service via infite loop can occur. An attacker can abuse the
```

Comment:
Could you explain more on what exactly a 'reload attempt' is for a list, and why that list is 'insufficient'

Reply:
Added better descriptions for reloading and insufficient lists